### ISSUE TITLE
Licensing is unclear

### ISSUE THREAD

**Issue by cassidyjames** [elementary/wallpapers#104]
Since we moved licensing to the deb-packaging branch, it's no longer clear what the licensing is for images in this repo or for any of the other code/metadata. A LICENSE.md file that clarifies GPLv3 for everything but the backgrounds folder, plus a link to https://github.com/elementary/wallpapers/blob/deb-packaging/debian/copyright for the backgrounds folder would probably suffice?

**Comment by danrabbit** [comment]
Yeah I’m uncertain why the licensing information is in the packaging actually

**Comment by MobileAZN** [comment]
I was looking into this and I think the reason that copyright file is there is for debian packaging standards itself: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

Maybe this file shouls also available in master so only 1 PR is needed?

This would solve a few open issues in this  repo. This one (#104) and the following:
https://github.com/elementary/wallpapers/issues/100
https://github.com/elementary/wallpapers/issues/157
